elrond-wasm/src/types/flags/code_metadata.rs

```diff
@@ -98,6 +107,41 @@ impl TypeAbi for CodeMetadata {
     }
 }
 
+impl SCDisplay for CodeMetadata {
+    fn fmt<F: FormatByteReceiver>(&self, f: &mut F) {
+        let mut first: bool = true;
+        if self.is_upgradeable() {
+            f.append_bytes(UPGRADEABLE_STRING);
+            first = false;
+        }
+        if self.is_readable() {
+            if !first {
+                f.append_bytes(b"|");
+            }
+            f.append_bytes(READABLE_STRING);
+        }
+        if self.is_payable() {
+            if !first {
+                f.append_bytes(b"|");
+            }
+            f.append_bytes(PAYABLE_STRING);
+        }
+        if self.is_payable_by_sc() {
+            if !first {
+                f.append_bytes(b"|");
+            }
+            f.append_bytes(PAYABLE_BY_SC_STRING);
+        }
+        if !(self.is_upgradeable()
+            | self.is_readable()
+            | self.is_payable()
+            | self.is_payable_by_sc())
+        {
+            f.append_bytes(DEFAULT_STRING);
+        }
```

Comment:
I think this `if` is the same as `if first { ... }`

Reply:
@dorin-iancu 
Nice catch! Thanks! I'd like to change the name of variable `first` to `metadata`. Any other ideas for the variable naming?